src/clients/cassandra/CassandraConnector.js

```diff
@@ -12,7 +12,12 @@ function executeBatchMutations(mutations) { // eslint-disable-line no-unused-var
   });
 }
 
-function executeQuery(query) { // eslint-disable-line no-unused-vars
+/**
+ * @param {string} query
+ * @param {string[]} params
+ * @returns {Promise.<object[]>}
+ */
+function executeQuery(query, params) { // eslint-disable-line no-unused-vars
```

Comment:
I agree this needs 2 parameters rather than passing in an unnecessary object with query and params properties.